### PR TITLE
willys/256818.Part6

### PR DESCRIPTION
**Pull request by wpschaub**
Respond to feedback from the engineering teams and our inspect and adapt workshops, that YAML pipelines are complex and require a steep learning curve. We need to reenforce the simplicity and lay the foundation for Said's app-type blueprints.

### CODE REVIEW

**Comment by wpschaub** [comment]
@solicitfeedback @said-akram-wcbbc @rbeuthin @604kev how can we improve this post? I want to re-emphasise that the blueprints make YAML pipelines a snap (if you keep it simple) and set the scene for @said-akram-wcbbc part 7, which emphasises that app-type blueprints make it even simpler.  THANK YOU for candid feedback!

**Comment by wsbctechnicalblog** [comment]
> @solicitfeedback @said-akram-wcbbc @rbeuthin @604kev how can we improve this post? I want to re-emphasise that the blueprints make YAML pipelines a snap (if you keep it simple) and set the scene for @said-akram-wcbbc part 7, which emphasises that app-type blueprints make it even simpler. THANK YOU for candid feedback!

This is another excellent post in the series of pipeline articles, @wpschaub. Congrats, awesome job (as always). Did I mention it reads exceptionally well? You could teach tech writing course :)

Not sure if this particular post is a good opportunity to emphasize infrastructure-as-code and the process of YAML pipelines reviews/approvals via PRs. We need to ween all engineers off any manual, touching with their bare hands GUI operations. EVERYTHING must be code which must be collectively reviewed before merging. I feel that, in general, computing infrastructure has been woefully neglected and left in a cowboy state (i.e. any authorized member can make any changes without others being notified until it's too late). As you know, that's not the case in software development, where every diff must be justified with tests and killed mutants (plus UAT). We need to start civilizing the infrastructure side in a similar fashion. Can't afford to continue in the Wild West cowboy style.

**Comment by wpschaub** [comment]
@solicitfeedback I can definitely inject the question of who owns these blueprints, which introduces the pull request workflow as well as the infrastructure as code concept. Will give it a whirl after snowshoeing tomorrow :)

**Comment by wpschaub** [comment]
@solicitfeedback here is the updated DRAFT, with a few questions at the end which will hopefully trigger the IaC/PR discussions. Please do not merge until @erwsch1 has reviewed and approved, I would also like @said-akram-wcbbc  and @rbeuthin  to give their "thumbs up".

I will work with Said on part 7 once this PR is merged and am also planning to start working on a related infrastructure-as-code (IaC) and PR workflow post.

**Review by erwsch1 — DISMISSED** [review]

File: content/moving-hundreds-of-pipeline-snowflakes-part6.md
Comment (said-akram-wcbbc) [inline content/moving-hundreds-of-pipeline-snowflakes-part6.md]: _discussed_ instead of **dscussed**

File: content/moving-hundreds-of-pipeline-snowflakes-part6.md
Comment (said-akram-wcbbc) [inline content/moving-hundreds-of-pipeline-snowflakes-part6.md]: replace **can can** by single _can_

Comment (wpschaub) [reply content/moving-hundreds-of-pipeline-snowflakes-part6.md]: Fixed

File: content/moving-hundreds-of-pipeline-snowflakes-part6.md
Comment (wpschaub) [reply content/moving-hundreds-of-pipeline-snowflakes-part6.md]: Fixed

**Review by said-akram-wcbbc — APPROVED** [review]

**Review by solicitfeedback — APPROVED** [review]

**Comment by solicitfeedback** [comment]
I reviewed and approved the diffs. The branch is now mergeable. Should I merge, or you need more reviewers to pitch in?

**Review by erwsch1 — APPROVED** [review]

**Comment by wpschaub** [comment]
@solicitfeedback @alexbunardzic go for launch!

**Pull request merged**
